Makefile

```
@@ -64,7 +64,7 @@ run-headless: | download-models update-submodules download-sample-videos
 	$(MAKE) run
 
 run-pipeline-server: | download-models update-submodules download-sample-videos build-pipeline-server
-	http_proxy= https_proxy= HTTP_PROXY= HTTPS_PROXY= RETAIL_USE_CASE_ROOT=$(RETAIL_USE_CASE_ROOT) docker compose -f src/pipeline-server/docker-compose.pipeline-server.yml up -d
+	RETAIL_USE_CASE_ROOT=$(RETAIL_USE_CASE_ROOT) docker compose -f src/pipeline-server/docker-compose.pipeline-server.yml up -d
```

should we include proxy envs?
```suggestion
	http_proxy=$(http_proxy) https_proxy=$(https_proxu) HTTP_PROXY=$(HTTP_PROXY) HTTPS_PROXY=$(HTTPS_PROXY)  RETAIL_USE_CASE_ROOT=$(RETAIL_USE_CASE_ROOT) docker compose -f src/pipeline-server/docker-compose.pipeline-server.yml up -d
```

---

This is no longer required with the docker-compose update. The build process also has these set.

---

gotcha